thin_egress_app/app.py

```diff
@@ -387,6 +387,16 @@ def get_user_from_token(token):
     return None
 
 
+@with_trace()
+def get_api_request_uuid(query_params):
+    api_request_uuid = None
+    if query_params is not None:
+        api_request_uuid = query_params.get("A-api-request-uuid")
+        if api_request_uuid is not None:
+            log.info("A-api-request-uuid query param is " + api_request_uuid)
+        return api_request_uuid
```

Comment:
`api_request_uuid` can just be local to the if statement.
```suggestion
    if query_params is not None:
        api_request_uuid = query_params.get("A-api-request-uuid")
        if api_request_uuid is not None:
            log.info("A-api-request-uuid query param is " + api_request_uuid)
        return api_request_uuid
        
    return None
```